.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,8 +26,8 @@ Environmental output
 - What openshift/kubernetes version are you running, and the architecture? 
 - `kubectl get pods -o wide -n < csi driver namespace> `
 - `kubectl get nodes -o wide`
-- CNSA/Spectrum Scale version 
-- Remote Spectrum Scale version 
+- CNSA/Storage Scale version 
```

Comment:
Even though internally we use "CNSA", for our external facing documentation, we try and avoid using this and it's not an official acronym.  Suggest making this two bullets.  Note the capitalization, container native is not capitalized, intentionally. 

- IBM Storage Scale container native version
- IBM Storage Scale version